.NET/Microsoft.Recognizers.Text/Culture.cs

```diff
@@ -5,6 +5,7 @@ namespace Microsoft.Recognizers.Text
 {
     public sealed class Culture
     {
+        public const string Arabic = "ar";
```

Comment:
Shouldn't this be a specific variant? Or "ar-*"?
The goals should be to cover as many variants as possible without needing separate models.

Reply:
Yes I discussed that with @hazemelh and decided to make it "ar-*" to refer to the Modern Standard Arabic.